JS.md

```diff
@@ -35,6 +35,19 @@ The following intrinsic objects are added:
 
 ### Function Properties of the `WebAssembly` object
 
+#### `WebAssembly.validate`
+
+The `validate` function has the signature:
+```
+Boolean validate(BufferSource bytes)
+```
+If the given `bytes` argument is not a
+[`BufferSource`](https://heycam.github.io/webidl/#common-BufferSource),
+then a `TypeError` is thrown.
+
+Otherwise, this function performs *validation* as defined by the WebAssembly
+spec and returns `true` if validation succeeded, `false` if validation failed.
```

Comment:
My thinking here is that `compile` isn't supposed to fail and, if it does, we can generate a big expensive error report (maybe eventually pretty-printing the text leading up to the validation failure, like we do in JS).  With `validate`, we do expect it to fail a lot (hence return value, not throwing), so I thought this should be super-fast and lightweight by design.  It seems like, if you are debugging bad compiler output, you have a wide variety of offline validator tools or `compile` at your disposal and so this might simply not be a use case for `validate`.